src/app/play/play-theme.module.css

```
@@ -1,20 +1,25 @@
 .theme {
+  min-height: 100svh;
+
+  /* Lock font tokens within /play to avoid route-CSS load order flashes */
+  --font-serif: ui-serif, Georgia, Cambria, "Times New Roman", Times, serif;
```

Stylelint (17.4.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)